src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.1.1-preview" />
+    <!-- <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.1.1-preview" /> -->
```

Comment:
@brjohnstmsft if you are re-recording why are you using an older version of resource manager nuget package.
You should be using the latest. I believe we did talk about it.
The implicit resource manager package is for legacy tests that are still using it. The sooner we move away from the legacy package the better it is.